lib/sftp.js

```diff
@@ -905,10 +911,44 @@ require("util").inherits(Sftp, Events.EventEmitter);
    * @type  {void}
    * @private
    */
-  function parseReply(data, i, cb) {
+  function parseReply(data) {
     // console.log("data: ", data, data.split("\n").length + " parts");
     this.emit("data", data);
 
+    // connection related messages
+    if (this.state & Sftp.STATE_CONNECTING) {
```

Comment:
im not sure if the whole if conditions in parseReply are correct, but if the tests are green, i dont care